cmd/zpool/zpool_vdev.c

```diff
@@ -721,8 +721,15 @@ make_leaf_vdev(nvlist_t *props, const char *arg, uint64_t is_log)
 		char *value = NULL;
 
 		if (nvlist_lookup_string(props,
-		    zpool_prop_to_name(ZPOOL_PROP_ASHIFT), &value) == 0)
+		    zpool_prop_to_name(ZPOOL_PROP_ASHIFT), &value) == 0) {
 			zfs_nicestrtonum(NULL, value, &ashift);
+			if (ashift != 0 && (ashift < 9 || ashift > 16)) {
```

Comment:
I didn't see them defined anywhere, but would it be worth having `ASHIFT_MIN` and `ASHIFT_MAX` defined in an appropriate header file? Thoughts?